Assets/Plugins/StatefulUI/Runtime/Core/StatefulComponent.cs

```diff
@@ -198,13 +198,13 @@ private GameObject GetObject(int role)
             return null;
         }
 
-        private Image GetImage(int role)
+        private ImageReference GetImage(int role)
```

Comment:
It's best to create a separate GetImageReference method to obtain an ImageReference instance.